docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -1002,6 +1002,16 @@ How to **investigate**:
   # Assuming KEDA is running in a dedicated namespace "keda":
   kubectl logs -n keda deployment/keda-operator-metrics-apiserver
   ```
+- Check KEDA operator logs
+  ```
+  # Assuming KEDA is running in a dedicated namespace "keda":
+  kubectl logs -n keda deployment/keda-operator
+  ```
+- Check that Prometheus is running (since KEDA depends on it)
```

Comment:
I would mention "by default" too, because you can override the metrics URL and you could even configure autoscaling to scrape metrics from Mimir itself.

```suggestion
- Check that Prometheus is running (since we configure KEDA to scrape custom metrics from it by default)
```